src/svm/impls/binary/binary.c

```diff
@@ -1555,31 +1555,35 @@ PetscErrorCode SVMLoadTrainingDataset_Binary(SVM svm,PetscViewer v)
 {
   MPI_Comm  comm;
 
-  Mat       Xt_training;
-  Mat       Xt_biased;
+  Mat       X_training;
+  Mat       X_biased;
   Vec       y_training;
 
   PetscReal bias;
   PetscInt  mod;
 
   PetscFunctionBegin;
   TRY( PetscObjectGetComm((PetscObject) svm,&comm) );
-  TRY( MatCreate(comm,&Xt_training) );
-  TRY( PetscObjectSetName((PetscObject) Xt_training,"Xt_training") );
+  /* Create matrix of training samples */
+  TRY( MatCreate(comm,&X_training) );
+  TRY( PetscObjectSetName((PetscObject) X_training,"X_training") );
+  TRY( PetscObjectSetOptionsPrefix((PetscObject) X_training,"X_training_") );
+  TRY( MatSetFromOptions(X_training) );
+
   TRY( VecCreate(comm,&y_training) );
   TRY( PetscObjectSetName((PetscObject) y_training,"y_training") );
 
-  TRY( SVMLoadDataset(svm,v,Xt_training,y_training) );
+  TRY( SVMLoadDataset(svm,v,X_training,y_training) );
```

Comment:
- [ ] It's not really documented but it seems that `SVMLoadDataset` loads Xt, not X.
- [ ] Aside, SVMLoadDataset doesn't seem to need the svm argument. Maybe it could be moved to utils.

Reply:
@haplav I've decided to change Xt -> X. I agree; it is not a good idea to have different naming related to similar objects in the project. I will change them in the whole project in the future. Good. Accepted. 
SVMLoadDataset will be moved to utils. Accepted.

Reply:
Regarding to the SVMLoadDataset function, it was renamed as PetscViewerLoadSVMDataset and moved to utils.